USAGE.md

```diff
@@ -8,100 +8,281 @@ Baritone commands can also by default be typed in the chatbox. However if you ma
 
 To disable direct chat control (with no prefix), turn off the `chatControl` setting. To disable chat control with the `#` prefix, turn off the `prefixControl` setting. In Impact, `.b` cannot be disabled. Be careful that you don't leave yourself with all control methods disabled (if you do, reset your settings by deleting the file `minecraft/baritone/settings.txt` and relaunching).
 
-# For Baritone 1.2.10+, 1.3.5+, 1.4.2+
+# Getting Help
 
-Lots of the commands have changed, BUT `#help` is improved vastly (its clickable! commands have tab completion! oh my!).
+Try `#help` - it's vastly improved with clickable commands and tab completion! Different help sections are available:
```

Comment:
Maybe just "it has clickable commands and tab completion"? I doubt anyone who needs this information knows what the old help command did.